src/main/scala/tech/cryptonomic/conseil/tezos/TezosTypes.scala

```diff
@@ -341,15 +342,12 @@ object TezosTypes {
     value: Option[PublicKeyHash]
   )
 
-  //represents unparsed micheline json
-  final case class AccountScript(code: String) extends AnyVal
-
   final case class Account(
     manager: PublicKeyHash,
     balance: scala.math.BigDecimal,
     spendable: Boolean,
     delegate: AccountDelegate,
-    script: Option[AccountScript],
+    script: Option[Contracts],
```

Comment:
Maybe I'm not familiar with what's inside but when can it fail to parse? Do you mean parsing from JSON to scala's case classes or parsing while converting from Micheline? In the second case, we should be safe because of error handling.

Reply:
what does the error handler do?

Reply:
Do you mean that at the bottom it's a string in the end anyway?

Reply:
> what does the error handler do?

I mean using ``Try`` on Michelson parsing

> Do you mean that at the bottom it's a string in the end anyway?

No, I'm trying to guess what you meant. :). Is there any way that one of the fields: ``code`` and ``storage`` will be missing?

Reply:
I'm not sure if there's any problem here, but this occurred to me
1. AccountScript is a simple wrapper of a String, which was not actually parsed originally (the code string)
2. Scripted.Contract contains a Micheline object, which itself is again a simple wrapper of strings, but this time I expect it to be the result of a parsed json result?

So I wondered, what happens when parsing fails, will we still have a valid Micheline instance to put in the Scripted.Contracts, which will in turn be set into the script attribute of the Account?